php-checks/src/test/resources/checks/DirectlyAccessingSuperGlobalCheck.php

```diff
@@ -4,3 +4,5 @@
 //   ^^^^^^
 
 $b = $_MY_ARRAY['name'];  // OK
+
+$_SESSION["newsession"]=$value; // OK  $_SESSION is a special case and can be accessed directly
```

Comment:
The new test case should look like the others so that we're confident that the exception works well, e.g.:
```
$c = $_SESSION['name'];
```